src/main/resources/templates/recipes.html

```diff
@@ -40,7 +40,7 @@ <h1> Przepisy </h1>
         <div class="dropdown">
             <form th:action="@{/recipes}">
                 <select name="meal">
-                    <option th:hidden="${meal}" th:value="${meal}" th:text="${meal.value}"></option>
+                    <option th:hidden="${meal}" th:value="${meal}" th:text="${meal}"></option>
```

Comment:
To trzeba wyjasnic czemu konieczna była taka zmiana - w samym filtrowaniu .value bylo celowe.